backend/helper_zero/views/donation.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from rest_framework import viewsets
+from rest_framework import status
+from rest_framework.response import Response
+
+from helper_zero.serializers import DonationSerializer
+from helper_zero.models import Donation, Organization
+
+class DonationView(viewsets.ModelViewSet):
+    serializer_class = DonationSerializer
+    queryset = Donation.objects.all()
+
+    def create(self, request):
+        request_dict = request.data
+        serializer = DonationSerializer(data=request_dict)
+        if serializer.is_valid():
+            org = Organization.objects.get(id=request_dict["org"])
+            if request_dict["status"] is None:
+                request_dict["status"] = "incomplete"
+            # TODO: may need to parse donation time start & end
+            donation = Donation(
+                org=org,
+                status=request_dict["status"],
```

Comment:
what are the possible values of status? wondering if this be calculated in the backend instead of in request_dict

Reply:
what do you mean by calculate it in the backend? I do agree we should probably limit what the possible statuses are. Not really sure what the python way to do it is...
this is just for donation creation though, which should just set it to incomplete when they get created. I'm assuming the front-end usually won't pass a status is.

Reply:
oh sorry i missed that you default it to 'incomplete' if there's no status

Reply:
just a nit but we can prolly move it out of request_dict since it's likely never going to be a post form param